app/javascript/listings/components/MessageModal.jsx

```diff
@@ -54,9 +57,12 @@ const MessageModal = ({
             <a href="/code-of-conduct">code of conduct</a>
           </em>
         ) : (
-          <em data-testid="generic-message-about-interactions" id="generic-message-about-interactions">
+          <em
+            data-testid="generic-message-about-interactions"
+            id="generic-message-about-interactions"
+          >
             Message must be relevant and on-topic with the listing. All private
-            interactions
+            interactions 
```

Comment:
Random observation, you seem to have some extra whitespace here 😃

Reply:
It is some git hook that is adding these spaces. Should I skip it?

Reply:
The git hook should be removing it. It runs `prettier` on the files you committed.

Reply:
I found that I had global prettier config that was causing this issue after I removed the global config everything was fine.